### PR TITLE
added filetypes

### PR DESCRIPTION
**Pull request by Reaxt**
just a few I think should be added,

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -3,20 +3,34 @@
...
     "video/mpeg",
-    "video/x-mpeg"
+    "video/x-mpeg",
+    "video/x-matroska",
+    "video/webm",
+    "application/x-7z-compressed",
+    "application/pdf",
+    "application/zip",
+    "application/x-sqlite"
+
```
Comment (deansheather) [inline config.json]: These newlines should be removed.

File: config.json [hunk 2]
```diff
@@ -3,20 +3,34 @@
...
     "image/gif",
     "image/png",
     "text/plain",
     "video/mp4",
     "video/mpeg",
-    "video/x-mpeg"
+    "video/x-mpeg",
+    "video/x-matroska",
+    "video/webm",
+    "application/x-7z-compressed",
```
Comment (deansheather) [inline config.json]: Not sure about compressed files, they allow for any filetype to be embedded.

File: config.json [hunk 3]
```diff
@@ -3,20 +3,34 @@
...
+    "application/pdf",
+    "application/zip",
```
Comment (deansheather) [inline config.json]: Read the comment about compressed files.

File: config.json (hunk 2 above)
Comment (deansheather) [reply config.json]: I say no for now, until we leave beta.

File: config.json [hunk 4]
```diff
@@ -3,20 +3,34 @@
...
   ],
   "maxFilesPerUpload": 3,
   "maxFileSize": 83886080,
   "textPlainExtensions": [
     "css",
     "html",
     "js",
-    "txt"
+    "txt",
+    "JSON",
```
Comment (deansheather) [inline config.json]: Capitals? Should be lowercase.

File: config.json [hunk 5]
```diff
@@ -3,20 +3,34 @@
...
+    "config",
```
Comment (deansheather) [inline config.json]: Replace `config` with `ini` and `yaml`.

File: config.json [hunk 6]
```diff
@@ -3,20 +3,34 @@
...
+    "audio/ogg",
     "image/jpeg",
     "image/gif",
     "image/png",
     "text/plain",
     "video/mp4",
     "video/mpeg",
-    "video/x-mpeg"
+    "video/x-mpeg",
+    "video/x-matroska",
```
Comment (deansheather) [inline config.json:15]: Should be accompanied by `audio/x-matroska`.

**Review by deansheather — CHANGES_REQUESTED** [review]

File: config.json (hunk 5 above)
Comment (Cynosphere) [reply config.json]: add `conf` and `cfg` too maybe, idk if they're as widely used as `ini` and `yaml` tho

Comment (deansheather) [reply config.json]: 👍

**Review by deansheather — APPROVED** [review]

**Pull request merged**